File: reddio-csharp/reddio.api/V1/Rest/GetRecordMessage.cs
using System.Text.Json.Serialization;

namespace Reddio.Api.V1.Rest;

public class GetRecordMessage
{
    public GetRecordMessage()
    {
    }

    public GetRecordMessage(string starkKey, long sequenceId)
    {
        StarkKey = starkKey;
        SequenceId = sequenceId;
    }

    [JsonPropertyName("stark_key")] public string StarkKey { get; set; }
    [JsonPropertyName("sequence_id")] public Int64 SequenceId { get; set; }
}

File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/GetRecordMessage.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class GetRecordMessage {
    @JsonProperty("stark_key")
    private String starkKey;
    @JsonProperty("sequence_id")
    private Long sequenceId;
}
